README.md

```diff
@@ -76,3 +76,9 @@ Songs:    completed    11s [============================================>] 100%
 
 Done! All rows have been deleted successfully.
 ```
+
+## Import as a Go package
+
+You can also make it done by calling `Run` function of `truncate` package by your own code in Go.
+
+It will help you to refer to [main.go](./main.go) in this repository.
```

Comment:
Maybe we can simply say:

```
You can also use spanner-truncate as a Go library from your Go application. The entry point is [Run](https://pkg.go.dev/github.com/cloudspannerecosystem/spanner-truncate/truncate#Run) function in `truncate` package.
```

Note that the Go doc is not available yet: https://pkg.go.dev/github.com/cloudspannerecosystem/spanner-truncate/truncate